service/src/main/kotlin/no/nav/su/se/bakover/service/klage/KlageServiceImpl.kt

```diff
@@ -66,6 +66,11 @@ class KlageServiceImpl(
             // TODO jah: Justere denne sjekken når vi har konseptet lukket klage.
             return KunneIkkeOppretteKlage.FinnesAlleredeEnÅpenKlage.left()
         }
+
+        if (sak.hentKlager().any { it.journalpostId == request.journalpostId }) {
```

Comment:
Kan flytte denne til en extension function i Klage.kt
```suggestion
        if (sak.klager.harJournalpostId(journalpostId) {
```